### DISCUSSION TITLE
Sample Database

### DISCUSSION THREAD

**Discussion by Goldstoned** [dbeaver/dbeaver#11318] [Q&A] [answered]
Hi

I am new to debeaver on a MAC. I am trying to find a local sample database for SQL Server to use. Please can someone point me in the right direction as to where I can find and load one?

Many Thanks

#### System information: 
- Operating system (distribution) and version
- DBeaver version

#### Connection specification:
- Database name and version

**Comment by uslss** [comment, marked as answer]
Hi @Goldstoned ,
DBeaver hasn't got local sample database for SQL Server.
You can find it here https://docs.microsoft.com/en-us/sql/samples/adventureworks-install-configure?view=sql-server-ver15&tabs=ssms
